src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -926,6 +927,11 @@ namespace OpenLoco::Ui::PromptBrowse
                     return;
             }
 
+            // Save last directory
+            fs::path lastDirectory = fs::u8path(&_directory[0]);
+            Config::getNew().last_save_path = lastDirectory.u8string();
+            Config::writeNewConfig();
```

Comment:
I think you should set the save game path as well at this point so that the current session changes where it is saving by default. A call to `Environment::resolvePaths` should do it.